packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -0,0 +1,119 @@
+import './index.scss';
+import { Desktop, Pattern, Touch } from '@carbon/pictograms-react';
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import ContentGroupPictograms from '../ContentGroupPictograms';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|Content Group Pictograms', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const pictograms = {
+      Desktop: 'Desktop',
+      Touch: 'Touch',
+      Pattern: 'Pattern',
+    };
+
+    const pictogram1 = select(
+      'Element 1 pictogram (required)',
+      pictograms,
+      pictograms.Desktop
+    );
+
+    const pictogram2 = select(
+      'Element 2 pictogram (required)',
+      pictograms,
+      pictograms.Touch
+    );
+
+    const pictogram3 = select(
+      'Element 3 pictogram (required)',
+      pictograms,
+      pictograms.Pattern
+    );
+
+    /**
+     * Returns the react component based on the value in the pictogram variables
+     *
+     * @param {string} sel string that defines the returning pictogram
+     * @returns {*} JSX pictogram component
+     */
+    const selectPictogram = sel => {
+      switch (sel) {
+        case 'Desktop':
+          return Desktop;
+        case 'Pattern':
+          return Pattern;
+        case 'Touch':
+          return Touch;
+      }
+    };
+
+    const heading = text(
+      'Pattern title (required)',
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+    );
+
+    const items = [
```

Comment:
Does it overcomplicate to just create a loop of the same data object, and maybe allow a way to add or remove pictogram items based on a count value?